newIDE/app/package.json

```diff
@@ -43,7 +43,7 @@
     "firebase": "^6.1.0",
     "fontfaceobserver": "2.0.13",
     "keen-tracking": "1.1.3",
-    "lodash": "4.17.4",
+    "lodash": "^4.17.15",
```

Comment:
Cancel these changes - and also cancel all changes in package-lock.json - as it's not related to your changes :)

Reply:
It might be got changed by mistake as I took a pull request from GDevelop's master. I will undo it.